schemas/hygiene/aep-hygiene-ops-record.schema.json

```diff
@@ -122,10 +129,201 @@
               },
               "meta:titleId": "aep-hygiene-ops##xdm:updateField##title##65571",
               "meta:descriptionId": "aep-hygiene-ops##xdm:updateField##description##2691"
+            },
+            "xdm:privacyDelete": {
+              "title": "Privacy Delete Request",
+              "description": "Represents data-subject requests to have their data deleted",
+              "type": "object",
+              "properties": {
+                "xdm:identities": {
+                  "title": "Set of identities",
+                  "description": "Set of identities whose data must be deleted",
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "additionalProperties": false,
```

Comment:
@perlmonger42 , let's remove all the additionalProperties instances in this PR as we do not have to explicitly set them as false because we should not allow additionalProperties anywhere outside map objects